### ISSUE TITLE
Wire lacks statuses for users such as busy, away and others

### ISSUE THREAD

**Issue by ddobrev** [wireapp/wire-desktop#900]
Wire Version: 2.15.2750
Wire for Web Version: 2017.08.24.1057
OS: macOS Sierra 10.12.6
Do you have an antivirus software installed: No

What steps will reproduce the problem?
1. Add a contact;
2. Have him move away for a long time or hibernate his computer.

What is the expected result?

The status of the contact changes to reflect he's away.

What happens instead?

There are no statuses at all.

Any instant messenger must provide statuses for users so that people on the other side know if they actually participate in a conversation.

**Comment by IpsmLorem** [comment]
That's kind of feature could be against privacy. User must be able to prevent sending of their status.

**Comment by Mediazept** [comment]
This feature request was discussed a several times. Eg [here ](https://github.com/wireapp/wire-desktop/issues/128 )or [here](https://github.com/wireapp/wire/issues/24). I also vote for an presence status option, at least for professional use.
I was looking for an alternative messenger solution for our company. Wire looks like it could replace our shitty Skype app. But I can't imagine how we could use a messenger as a team without knowing the presence status of each other.
I understand the privacy concerns, but in a team environment that doesn't make sense. I need to show that I'm not available (at the phone with a customer or in a meeting). I don't want to receive messenges when I'm busy. And I want to know if my colleague is available for an urgent chat or call.
I still do not understand why Wire doesn't give this option. It could be a individual per user setting like in Skype or even better a setting for a certain group of people.
Even in private use I would like that my friends or family can see my status. If I decide to hide it, I can switch to "show me offline". That's an easy solution with full privacy control.
I doubt that Wire can become a successful tool for teams without this feature. 
So, to sum it up: I strongly vote for a presence indicator which is fully user controllable.

**Comment by ddobrev** [comment]
I completely agree with @Mediazept and I am actually surprised there can be a messenger in existence without such a basic feature. I don't understand how privacy can be a concern as long as there can be a simple option not to show it. I understand even less why the two issues @Mediazept 's linked have been closed without any kind of resolution.

**Comment by gregor** [comment]
@ddobrev Thank you for your concern. While I can relate to your feature request it is not something that is currently on the roadmap. The nature of the discussed feature is such that it would have to be addressed by all client teams - please consider the implementation effort. While we continue to re-eavluate our priorities regarding feature developement, a presence indicator is not on our immediate roadmap. There are certainly many ways to implement such a feature while keeping privacy in mind - but the needed effort is not something we are going to commit to at this point in time.

We like to keep our issues actual for bugs and problems. Feature requests can always be filed on our main repository https://github.com/wireapp/wire/issues or brought to the attention of our customer support.

Therefore the previous feature requests on this repo have been closed - and I will go ahead and do the same with this one.

**Comment by Mediazept** [comment]
@gregor: You should check your links. At the [help page](https://support.wire.com/hc/de/requests/new) there's a link "if you want to give us feedback" which directs users to [this issues tracker](https://github.com/wireapp/wire-desktop/issues). It seems like this was not intended by the devs ;)

Regarding this issue: I was really enthusiastic when I learned about Wire and it's features. It's a pity that we can't use it. So, back to start, searching a reliable messenger for our teams.

**Comment by gregor** [comment]
Thanks for the feedback. I will forward it internally to reword the support page to more precisely refer to problems and issues instead of general feedback and feature requests towards Wire as a platform which are easier to track via our customer support channels and our main repository. This does not mean that we agree on the importance of a request or that it will actually get implemented either.
